rikai/public/templates/dist/js/addTask.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function(){
+	$.ajaxSetup({
+	    headers:
+	    {'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')}
+	});
+	$("#AddTask").submit(function(e) {
+		e.preventDefault()
+		var route = $(this).attr('action')
+		var name = $(this).find('input[name="name"]').val()
+		var detail = $(this).find('textarea[name="detail"]').val()
+		console.log(name);
+		console.log(detail);
+		if(name=='' || detail=='' ){
+			alert('Bạn chưa nhập tên Task hoặc chi tiết')
```

Comment:
Đưa i18n vào nhé
https://itzone.com.vn/en/article/how-to-use-localization-in-js-with-the-laravel-project/
https://viblo.asia/p/cach-dung-localization-trong-js-voi-project-laravel-part-1-naQZRvkjZvx?__cf_chl_jschl_tk__=de24145ecc02615d0b5deca5e03fe580badebc73-1625546031-0-AYQwBBLp5GRqNfNkIMQ0SQ0GwjtE5mtPjmdVE4Vfv5Q4kgBuiVdbTQozmKM654pM9V0s-6Sj_O7xqtqwKAsruqc9CvMD76rdllPCRNfI_l4MbpX_UcqXc8lweuk09CAgNNs-HZQA5nbaAE86pgI5aizstOZMo-ouOQK33FtmuVb7XqfeM5j42nMZhVRCmn-JZuoWc1khpH6-IPDSQkUrqHP18WuLZmIcw0msh2LL6sqzH8MJFUTsAC7rbq1X81boCPeFdCArO-IW_Owx0lTgIA43kegSntjUiLKg1cx7PBKXy52yY0aVjEoH0iJPfmHxNBYUs6O2qejtbJUn8-bR_r9mol_gchfGBSlBZfH3k9fw-fpxA7nZGeq9k5Yk4WEkc2CRFPsdd4tyG8SwqF6Lm_nwymg4fx9kkfIFmo9TzzmxguPFhJPirQH7Mx-jGEPmwSJyknqbnRAcO7IsjVfFEOxzTOj4w7tszAHZXw9d3buw